src/completion/default.rs

```diff
@@ -75,6 +75,10 @@ impl Completer for DefaultCompleter {
         // `only_buffer_difference` is false
         let line = if line.len() > pos { &line[..pos] } else { line };
         if !line.is_empty() {
+            // When editing a multiline buffer, there can be new line characters in it.
+            // Also, by replacing the new line character with a space, the insert
+            // position is maintain in the line buffer.
+            let line = line.replace("\r\n", " ").replace('\n', " ");
```

Comment:
```suggestion
            let line = line.replace('\r', " ").replace('\n', " ");
```
I misread the [original code](https://github.com/nushell/reedline/blob/main/src/menu/columnar_menu.rs#L551), `\r\n` is supposed to be replaced with two spaces rather than one, sorry about that

Reply:
thanks for catching that!